addons/advanced_throwing/functions/fnc_drawArc.sqf

```diff
@@ -50,10 +50,10 @@ for "_i" from 0.05 to 1.45 step 0.1 do {
             _cross = 1; // 1: Distance Limit (Green)
         } else {
             // Even vanilla throwables go through glass, only "GEOM" LOD will stop it but that will also stop it when there is glass in a window
-            if (lineIntersects [_prevTrajASL, _newTrajASL]) then { // Checks the "VIEW" LOD
+            if !((lineIntersectsSurfaces [_prevTrajASL, _newTrajASL]) isEqualTo []) then { // Checks the "VIEW" LOD
```

Comment:
I am fairly sure that was on purpose. Comments confirm.

If we can't "view" through it, it can't go through (LI check), mark red. Then if GEOM or FIRE goes through, mark yellow (LIS check).

Reply:
Reverted for the moment.